.github/workflows/release_build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           image_uri_with_tag: "adot-autoinstrumentation-python:test"
           push_image: false
-          load_image: false
+          load_image: true
```

Comment:
This change comes from the fix that we did on the `release/v0.1.x` branch: https://github.com/aws-observability/aws-otel-python-instrumentation/pull/160

Reply:
was about to ask, thanks for call out

Reply:
Lets cut a backlog item that asserts that either `push_image` or `load_image` is true.

Reply:
done